internal/locale/locales/en-us.yaml

```diff
@@ -1868,7 +1868,7 @@ err_update_corrupt_install:
     Uninstallation instructions: {{.V0}}/troubleshooting/uninstall/
     Installation instructions: {{.V0}}/install/
 flag_state_use_path_description:
-  other: Where to install the project. The default location is in your ~/Projects directory
+  other: Where to install the project. The default location is in the {{.V0}} directory
```

Comment:
If the value of the project directory is changed in the config then is it still the default location? I think we could update this message to just say the current location.

Also, do we want to communicate to users how to change this value? Or is that something to be done at a later date?

Reply:
When the project directory is changed in the config, this string will specify that newly configured location. Is that what you meant?

As far as communicating how to change this value, I presumed this would be in some sort of centralized documentation for all of our configuration options that has not yet been written. I suppose it might be a good idea to mention it there though.

Reply:
What I mean is that the default is `~/Projects` but if I change that then this messaging says the default is whatever I changed it to. I admit that I'm being nit-picky here 😛

Reply:
Yes, whatever they changed it to becomes the "new default" with respect to this `--path` argument that's being documented. You are correct that it's not the "true" default in terms of out-of-box experience, but then there's always `state clean config`, right? :)